local_ai_server/server.py

```
@@ -4635,8 +4676,131 @@ async def _handle_llm_tool_request(
+    ) -> None:
+        call_id = data.get("call_id")
+        if call_id:
+            session.call_id = call_id
+        request_id = str(data.get("request_id") or "").strip() or None
+        tool_name = str(data.get("tool_name") or data.get("function_call_id") or "tool").strip()
+        result = data.get("result") if isinstance(data.get("result"), dict) else {"value": data.get("result")}
+        is_error = bool(data.get("is_error"))
+
+        try:
+            result_json = json.dumps(result or {}, ensure_ascii=False, default=str)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)